server/@types/hmpo-form-wizard/index.d.ts

```diff
@@ -121,20 +132,27 @@ declare module 'hmpo-form-wizard' {
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       validateFields(req: FormWizard.Request, res: Express.Response, callback: (errors: any) => void)
 
-      locals(req: Request, res: Express.Response, next: Express.NextFunction): Promise
+      // eslint-disable-next-line no-underscore-dangle
+      _locals(req: Request, res: Express.Response, next: Express.NextFunction): Promise
+
+      locals(req: Request, res: Express.Response, next: Express.NextFunction): object
```

Comment:
Why there are "two" locals now?

Reply:
So these are built in functions within the form wizard. From what I can tell and understand, the `_locals` is used when there are async processes going on within it, hence the promise at the end, whereas the normal `locals` is simpler and is used when you don't need async processes within it.